nimble/host/audio/services/bass/include/services/bass/ble_audio_svc_bass.h

```diff
@@ -324,6 +324,10 @@ struct ble_svc_audio_bass_operation {
             /** Number of subgroups */
             uint16_t num_subgroups;
 
+            /** Subgroup entries */
+            struct ble_svc_audio_bass_subgroup
+                subgroups[BLE_SVC_AUDIO_BASS_SUB_NUM_MAX];
+
             /** BIS Synchronisation of subgroups */
             uint32_t bis_sync[BLE_SVC_AUDIO_BASS_SUB_NUM_MAX];
```

Comment:
this one should not be needed, as you can pass the bis sync in subgroups[i].bis_sync instead

Reply:
LGTY now?